### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by Cavonstavant**

### CODE REVIEW

File: tests/unit_tests/Parser/Parser_test.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*
+** EPITECH PROJECT, 2022
+** plazza
+** File description:
+** Parser_test
+*/
```
Comment (LucasTesnier) [inline tests/unit_tests/Parser/Parser_test.cpp:6]: Empty file ?

File: tests/unit_tests/MessageQueue/MessageQueue_test.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*
```
Comment (LucasTesnier) [inline tests/unit_tests/MessageQueue/MessageQueue_test.cpp:1]: .ccp file

**Review by LucasTesnier — APPROVED** [review]
NTR

**Review by osvegn — APPROVED** [review]
LGTM

**Pull request merged**